### DISCUSSION TITLE
Sending email notification without sender login and password

### DISCUSSION THREAD

**Discussion by bluinchiostro** [great-expectations/great_expectations#4892] [General]
I would like to send failure emails to my address, without a sender email account.
In Python it's possible importing email.message.EmailMessage and simply send plain text email messages over localhost to the email address I specify. In GE the "sender_login" and "sender_password" are both required. It would be very useful to test this feature using an email relay and without sender login e password.

**Comment by AFineDayFor** [comment]
Howdy @bluinchiostro, I'm moving this to Discussions.

You would be able to leverage an `action`, you can read about [custom actions](https://docs.greatexpectations.io/docs/terms/action/#how-to-create) and let us know if that might be in the direction you want.

**Comment by bluinchiostro** [comment]
Hi @AFineDayFor, yes, I'd like to configure an EmailAction but I'd like to keep the parameters sender_login and sender_password empty (https://greatexpectationstalk.slack.com/archives/CUUMYDJAX/p1645051416909119). Is it possibile?
